src/queries.rs

```diff
@@ -5,17 +5,17 @@ use crate::msg::QueryMsg;
 #[entry_point]
 pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
     match msg {
-        QueryMsg::GetOwners {} => to_binary(&query::get_owners(deps)?),
+        QueryMsg::GetConfig {} => to_binary(&query::get_config(deps)?),
```

Comment:
```suggestion
        QueryMsg::GetConfig {} => to_binary(&CONFIG.load(deps.storage)?),
```
Overkill for nothing.